linkml_runtime/utils/namespaces.py

```diff
@@ -192,7 +192,9 @@ def uri_or_curie_for(self, prefix: Union[str, URIRef], suffix: str) -> str:
                     return k + ':' + suffix
             return self.join(str(prefix), suffix)
         elif prefix not in self:
-            raise ValueError(f"Unrecognized prefix: {prefix}")
+            # raise ValueError(f"Unrecognized prefix: {prefix}")
+            print(f"WARNING: Unrecognized prefix: {prefix}")
```

Comment:
I would prefer logging.error or something that writes on stderr

Reply:
Yes, this is not ideal solution - I'm happy to close or change the PR while I investigate #96 further to be honest.